DigitalLearningSolutions.Data/DataServices/UserDataService/AdminUserDataService.cs

```diff
@@ -92,6 +93,41 @@ public void UpdateAdminUser(string firstName, string surname, string email, byte
             );
         }
 
+        public void UpdateAdminAccount(AdminRegistrationModel model, int adminId)

```

Comment:
Using the admin registration model for this seems a bit strange when we don't need a large portion of the details that it has and we are doing an update not a registration. Might be worth making a new model for this?